Bottom: 5640abebbeb803875d6f5a37194a424414f92b18
Top:    fcd88b07540162340780ccb36cbd6b92160c3aa8
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-03 16:37:07 -0500

Refresh of add-vmdaedalus-inventory

---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index 59b6eea4..c1c2be69 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -13,8 +13,8 @@
       src: /tmp/ansible_variables
       dest: "{{inventory_hostname}}_ansible_variables"
 
-- name: Configure daedalus partitioning
-  hosts: daedalus
+- name: Symlink on-disk copies of bacula archives
+  hosts: bacula
   remote_user: ansibleremote
   become: true
   become_user: root
